english/first-contact/first-contact.md

```diff
@@ -0,0 +1 @@
+Because you have established the marshmallow creature is a sentient being, you begin the official Star Fleet First Contact procedures.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.